moirae/estimators/online/utils/model.py

```diff
@@ -45,7 +45,11 @@ def __init__(self,
                  cell_model: CellModel,
                  asoh: HealthVariable,
                  transients: GeneralContainer,
-                 inputs: InputQuantities):
+                 inputs: InputQuantities,
+                 converters: Optional[ModelWrapperConverters] = ModelWrapperConverters.defaults()):

```

Comment:
Another "Optional" that cannot be None. I'll fix it before merging

Reply:
oh sorry, I guess I missed this one... thanks!